### PR TITLE
Fix mark as read in the activity center

### LINKED ISSUE #3011: Can't mark as read the very last message (either from activity center or through the Mark as read chat option)
Having a community channel with several unread messages (where u was mentioned or your message was replied on), mark the messages as read with chat option or through Activity Center - mark as read. As result, the very last unread can't be marked as read:

`DBG 2021-07-21 23:12:05+03:00 callPrivateRPC                             topics="rpc" tid=7844702 file=core.nim:20 rpc_method=eth_call
qrc:/app/AppLayouts/Chat/ChatColumn/ChatComponents/ActivityCenterMessageComponent.qml:41: Error: Insufficient arguments`

https://user-images.githubusercontent.com/82375995/126553949-e3c7f7ca-71fc-409b-9175-d520db2b0838.mov

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #3011

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/InvitationBubble.qml [hunk 1]
```diff
@@ -18,11 +18,22 @@ Item {
     width: rectangleBubbleLoader.width
 
     function getCommunity() {
-        let community = JSON.parse(chatsModel.communities.list.getCommunityByIdJson(communityId));
```
Comment (alaibe) [inline ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/InvitationBubble.qml:21]: Why are we using json here?

**Review by alaibe — APPROVED** [review]

File: ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/InvitationBubble.qml (hunk 1 above)
Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/InvitationBubble.qml:21]: TBH, I don't remember. I didn't code the original

**Review by iurimatias — CHANGES_REQUESTED** [review]
I tried to replicate this so I could test the fix, but the mentions don't show in the activity center

Screen Shot 2021-07-23 at 4 42 05 PM

**Comment by iurimatias** [comment]
> I tried to replicate this so I could test the fix, but the mentions don't show in the activity center
> 
> Screen Shot 2021-07-23 at 4 42 05 PM

so it seems this is expected due to spam reasons and what the status-go api returns. it will only show in the activity center for mentions & replies on group chats and communities

**Pull request merged**